graylog2-web-interface/src/integrations/aws/common/FormWrap.tsx

```diff
@@ -46,10 +61,10 @@ const ErrorOutput = styled.span`
   display: block;
 `;
 
-const ErrorToggleInfo = styled.button`
+const ErrorToggleInfo = styled.button<{ isDarkMode: boolean }>`
   border: 0;
   background: none;
-  color: #1f1f1f;
+  color:${({ isDarkMode }) => (isDarkMode ? 'white' : 'black')};
```

Comment:
@danotorrey @kingzacko1 @ryan-carroll-graylog Just some feedback for future changes.

1. In styled components we do not check if the dark mode is active, to display a different color. We point to a specific color in the theme `theme.colors.text.primary` and the theme will provide different colors depending on the selected theme mode (dark / light).
2. We avoid hard coded colors and only point to the theme, to be able to customize the colors easily.
3. When you define a variable for a styled component like `isDarkMode` it should to be prefix with a `$`, otherwise this will create an attribute for the DOM node. https://styled-components.com/docs/api#transient-props